r/R/utilities_expression.R

```diff
@@ -0,0 +1,62 @@
+#' Returns expression values for selected genes
+#'
+#' @param genes - Must have names and input(ensmbl ids)
+#' @param data
+#'
+#' @return
+#' @export
+#'
+#' @examples
+getExpressionValues <- function(genes, data) {
+  quantile_threshold <- 0.95
+  #
+  # Get the expression values for those genes in the corresponding matrix.
+  geneExpression <- data@assays$RNA@data[unique(genes$input), , drop = FALSE]
```

Comment:
why do we have this `,  ,` empty space between commas here? It looks easy to omit and delete in the future. Is there some parameter that we can set there to explicitly indicate that we don't want whatever we don't want to happen?

Reply:
It's common R syntax. You wouldn't delete that if you know what you are doing because then you are not selecting columns from that data frame